unity-renderer/Assets/Scripts/Tests/ValidationTests/SettingAssetsValidationTests.cs

```diff
@@ -22,5 +22,21 @@ public void QualitySettingsAntiAliasingShouldBeZero()
 
             Assert.That(antiAliasingValue, Is.EqualTo("0"));
         }
+
+        [Test]
+        public void SplashScreenShouldBeFalse()
+        {
+            string qualityAssetPath = Application.dataPath.Remove(Application.dataPath.Length - "Assets".Length)
+                                      + "ProjectSettings/ProjectSettings.asset";
+
+            string antiAliasingValue =
```

Comment:
I think variable names here need a bit of love

Reply:
oopsie, fixed